src/main/resources/reference.conf

```diff
@@ -7,6 +7,18 @@ ensembl_version = "YYY"
 efo_version = "MM-mm-pp"
 // --- END --- Release baseline parameters --- //
 
+// Bucket name information
```

Comment:
the config keys have been moved to the literature section and the pathBaseOutputLiterature has been removed because it was only being used in the output key that was previously removed